src/components/CalendarDay.jsx

```diff
@@ -174,6 +174,7 @@ class CalendarDay extends React.Component {
 CalendarDay.propTypes = propTypes;
 CalendarDay.defaultProps = defaultProps;
 
+export { CalendarDay as PureCalendarDay };
```

Comment:
Could Pure be a misleading term here? Maybe overloaded

Reply:
wait, why are we exporting this at all? best practice is to only use `.dive()` to get at the unwrapped component

Reply:
because we are spying on prototype methods. `dive` doesn't solve that issue afaik, does it?

Reply:
you have me there.

any way we could avoid the spying?

it'd be ugly but we could `const Pure = shallow(<Wrapped />).dive().type(); sinon.spy(Pure.prototype, 'foo')`